\hypertarget{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1NewConnectionError}{}\section{pip.\+\_\+vendor.\+urllib3.\+exceptions.\+New\+Connection\+Error Class Reference}
\label{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1NewConnectionError}\index{pip.\+\_\+vendor.\+urllib3.\+exceptions.\+New\+Connection\+Error@{pip.\+\_\+vendor.\+urllib3.\+exceptions.\+New\+Connection\+Error}}


Inheritance diagram for pip.\+\_\+vendor.\+urllib3.\+exceptions.\+New\+Connection\+Error\+:
\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=350pt]{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1NewConnectionError__inherit__graph}
\end{center}
\end{figure}


Collaboration diagram for pip.\+\_\+vendor.\+urllib3.\+exceptions.\+New\+Connection\+Error\+:
\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=350pt]{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1NewConnectionError__coll__graph}
\end{center}
\end{figure}
\subsection*{Additional Inherited Members}


\subsection{Detailed Description}
\begin{DoxyVerb}Raised when we fail to establish a new connection. Usually ECONNREFUSED.\end{DoxyVerb}
 

The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
venv/lib/python3.\+9/site-\/packages/pip/\+\_\+vendor/urllib3/\hyperlink{pip_2__vendor_2urllib3_2exceptions_8py}{exceptions.\+py}\end{DoxyCompactItemize}
